packages/webapp/src/pages/sessions.tsx

```diff
@@ -0,0 +1,97 @@
+import { hexToUint8Array } from "eosjs/dist/eosjs-serialize";
```

Comment:
this is an example page as a reference of how to actually create and sign with sessions.